### DISCUSSION TITLE
[백준 4344] 평균은 넘겠지

### DISCUSSION THREAD

**Discussion by BanApp** [ghdcksgml1/Algorithm_Study#83] [1일 1알고리즘]
<img width="1218" alt="스크린샷 2022-02-26 오후 2 34 42" src="https://user-images.githubusercontent.com/93313445/155830671-ce5ee439-fa6e-494d-8f18-66ee1c679514.png">

```
import sys

n = int(sys.stdin.readline())

for i in range(n):
    arr = list(map(int, sys.stdin.readline().split()))
    k = arr[0]
    a = sum(arr[1:]) / k
    cnt = 0

    for j in range(1,k+1):
        if arr[j] > a:
            cnt = cnt+1

    ans = (cnt/k)*100

    print(format(ans,".3f") + '%')
```
arr[0]을 기준으로 삼으면 되는 문제
출력할때 소수점 3자리 까지 표현되도록 format 함수를 썼다.
round 함수는 반올림을 해주기 때문에 여기서는 쓰면안된다.
이정도 풀었으면 평균은 넘겠지?  \\^ㅇ^/

**Comment by ghdcksgml1** [comment]
알고리즘 시작한 순간 평균은 넘은겁니다. ㅋㅋㅋㅋㅋ
